### DISCUSSION TITLE
Import from an old Foswiki

### DISCUSSION THREAD

**Discussion by tiagobugarin** [requarks/wiki#3191] [Help / Questions] [answered]
I have a wiki running a very old Foswiki installation (1.1.9) over a very old Debian server (7.11). This wiki is full of data. I am trying to figure out a way to move that trove of information to an updated system.

How would it be possible to import all Foswiki data to a clean installation of Wiki.js?

**Comment by NGPixel** [comment, marked as answer]
Check if you can export your content as markdown or HTML files? If yes, then you can import them via the Storage modules. Otherwise you're only option is to copy-paste content manually.
